modules/xcm-bridge-hub-router/src/lib.rs

```diff
@@ -87,16 +93,14 @@ pub mod pallet {
 		/// Also used for filtering `Bridges` by `BridgedNetworkId`.
 		/// If not specified, allows all networks pass through.
 		type BridgedNetworkId: Get<NetworkId>;
-		/// Configuration for supported **bridged networks/locations** with **bridge location** and
-		/// **possible fee**. Allows to externalize better control over allowed **bridged
-		/// networks/locations**.
-		type Bridges: ExporterFor;
```

Comment:
I've removed this configuration - now the pallet has the separate `Bridges` **storage map** for that. Also - `BaseFee` is returned because of that. It solves the #2380

Reply:
In other words, "opening the bridge" would now mean: (1) inserting an entry to the storage map (2) calling the `open_bridge` method of `pallet-xcm-bridge-hub` at sibling/child bridge hub. Right now it needs to be done using governance, but we may add some helper functions for that later